gui-libs/vte/vte-0.76.3.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit gnome.org meson python-any-r1 vala xdg
+
+DESCRIPTION="Library providing a virtual terminal emulator widget"
+HOMEPAGE="https://wiki.gnome.org/Apps/Terminal/VTE"
+
+SRC_URI="${SRC_URI} !vanilla? ( https://dev.gentoo.org/~mattst88/distfiles/${PN}-0.74.0-command-notify.patch.xz )"
+# Once SIXEL support ships (0.66 or later), might need xterm license (but code might be considered upgraded to LGPL-3+)
+LICENSE="LGPL-3+ GPL-3+"
+SLOT="2.91-gtk4" # vte_api_version + "-gtk4" in meson.build
+
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="+crypt debug gtk-doc +icu +introspection ptyxis systemd +vala vanilla"
+REQUIRED_USE="
+	gtk-doc? ( introspection )
+	vala? ( introspection )
+"
+
+DEPEND="
+	>=gui-libs/gtk-4.14:4[introspection?]
+	>=x11-libs/cairo-1.0
+	>=dev-libs/fribidi-1.0.0
+	>=dev-libs/glib-2.72:2
+	crypt?  ( >=net-libs/gnutls-3.2.7:0= )
+	icu? ( dev-libs/icu:= )
+	>=x11-libs/pango-1.22.0
+	>=dev-libs/libpcre2-10.21:=
+	systemd? ( >=sys-apps/systemd-220:= )
+	>=app-arch/lz4-1.9
+	introspection? ( >=dev-libs/gobject-introspection-1.56:= )
+	x11-libs/pango[introspection?]
+"
+RDEPEND="${DEPEND}
+	~gui-libs/vte-common-${PV}[systemd?]
+"
+BDEPEND="
+	${PYTHON_DEPS}
+	dev-libs/libxml2:2
+	dev-util/glib-utils
+	gtk-doc? ( dev-util/gi-docgen )
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+	vala? ( $(vala_depend) )
+"
+
+src_prepare() {
+	default
+	use vala && vala_setup
+	xdg_environment_reset
+
+	use elibc_musl && eapply "${FILESDIR}"/${PN}-0.66.2-musl-W_EXITCODE.patch
+
+	if ! use vanilla; then
+		# From https://src.fedoraproject.org/rpms/vte291/raw/rawhide/f/vte291-cntnr-precmd-preexec-scroll.patch
+		# Adds OSC 777 support for desktop notifications in gnome-terminal or elsewhere
+		eapply "${WORKDIR}"/${PN}-0.74.0-command-notify.patch
+	fi
+
+	if use ptyxis; then
```

Comment:
Why don't apply them when USE vanilla is disabled? I would also add a note explaining that they come from Fedora (to also control better the updates/fixes that they could have). Maybe we can simply list the same references as they do:
https://src.fedoraproject.org/rpms/vte291/blob/f40/f/vte291.spec#_24

Also, aren't those patches replacing the vte-0.74.0-command-notify.patch one?

Reply:
Done in https://github.com/gentoo/gentoo/pull/34844/commits/4823224d61b12fd97a25efcf977885dc04d7904f